docs/script.rst

```diff
@@ -386,8 +382,8 @@ Applying some syntax sugar, you can do the same in just one line of code::
 Removing part of a string
 -------------------------
 
-You can subtract a part of a String value using a regular expression pattern. The first match found is
-replaced with an empty String. For example::
+You can remove part of a ``String`` value using a regular expression pattern. The first match found is
```

Comment:
"Subtract" sounds weird except in an arithmetic (or very specific technical) context in English.